### PR TITLE
Feature/statistics

### PR DESCRIPTION
**Pull request by StephieTack**

### CODE REVIEW

File: components/statistic/StatisticCircles.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import styled, { keyframes } from "styled-components";
+
+export default function StatisticCircles({ count, maxCount, color, filter }) {
+  const maxRadius = 100;
+  const radius = (Math.sqrt(count) / Math.sqrt(maxCount)) * maxRadius;
```
Comment (F-Kirchhoff) [inline components/statistic/StatisticCircles.js:5]: ✨

File: components/statistic/StatisticCircles.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  const animationKey = `${filter}-${count}`;
+
+  return <Circle key={animationKey} size={radius * 2} color={color} />;
```
Comment (F-Kirchhoff) [inline components/statistic/StatisticCircles.js]: please prefix styled component props with a `$`

File: components/statistic/StatisticList.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  const filteredEmotions =
+    customDateRange.start && customDateRange.end
+      ? emotions.filter((emotion) => {
+          const emotionDate = new Date(emotion.dateTime);
+          return (
+            emotionDate >= customDateRange.start &&
+            emotionDate <= customDateRange.end
+          );
+        })
+      : emotions;
```
Comment (F-Kirchhoff) [inline components/statistic/StatisticList.js]: you can move this logic into your statistic page before rendering this component. then you can pass filteredEmtions to `StatisticList`. Then you don't need to pass down the customDateRange:

```js
// pages/statistic/index.js

  const filteredEmotions =
    customDateRange.start && customDateRange.end
      ? emotions.filter((emotion) => {
          const emotionDate = new Date(emotion.dateTime);
          return (
            emotionDate >= customDateRange.start &&
            emotionDate <= customDateRange.end
          );
        })
      : emotions;

return ...
   <StatisticList emtions={filteredEmotions} emotionTypes={emotionTypes}/> // no customDateRange needed here

```

**Review by F-Kirchhoff — APPROVED** [review]
Hey Stephie,
great PR! This was a hard US and the final code looks great! I added one small optional suggestion, but please feel free to ignore it.

File: components/statistic/StatisticList.js (hunk 3 above)
Comment (StephieTack) [reply components/statistic/StatisticList.js]: Yes, of course! Good move. 👯

**Pull request merged**